### DISCUSSION TITLE
How can I implement cylinder geometry

### DISCUSSION THREAD

**Discussion by jacktang** [lululxvi/deepxde#684] [General]
Hello,

First of all, thanks for the great project! 
I've read deepXDE documents, and don't find any resource on user-define-geometry. I going to define cylinder geometry. Just follow the [Cuboid source code](https://github.com/lululxvi/deepxde/blob/70c5c1353df4f6a57efd31eca558b8f29b7265ec/deepxde/geometry/geometry_3d.py) is ok?

**Comment by lululxvi** [comment]
You need to implement all the abstract methods of the base class https://github.com/lululxvi/deepxde/blob/master/deepxde/geometry/geometry.py
